pkg/workflow_template.go

```diff
@@ -334,6 +388,22 @@ func (c *Client) listWorkflowTemplateVersions(namespace, uid string) (workflowTe
 	return
 }
 
+func (c *Client) listWorkflowTemplateVersionsDBAll(paginator *pagination.PaginationRequest) (workflowTemplateVersions []*WorkflowTemplateVersion, err error) {
+	return c.selectAllWorkflowTemplateVersionsDB(paginator)
+}
+
+// listWorkflowTemplateVersionsDB grabs WorkflowTemplateVersions from the database.
+// This is a workaround for listWorkflowTemplateVersions until it is refactored.
+// See https://github.com/onepanelio/core/issues/436
+func (c *Client) listWorkflowTemplateVersionsDB(namespace, uid string) (workflowTemplateVersions []*WorkflowTemplateVersion, err error) {
+	dbVersions, err := c.selectWorkflowTemplateVersionsDB(namespace, uid)
+	if err != nil {
+		return nil, err
+	}
+
+	return dbVersions, err
+}
+
```

Comment:
@aleksandrmelnikov Do we need this method? Why don't we just call
```golang
selectWorkflowTemplateVersionsDB
```

Reply:
Agreed